src/components/general/Navbar/NavMenu/index.js

```diff
@@ -26,7 +26,6 @@ const NavItem = ({ itemKey, className, disabled, ...otherProps }) => (
         to={reverseRoute(pathNames[itemKey], otherProps)}
         className={`${className} ${disabled ? styles.disabled : ''}`}
         disabled={disabled}
-        exact
```

Comment:
Why was this removed?

Reply:
It's just for visual change. (The item will be active, even when the child is matched)